app/src/main/java/org/jellyfin/androidtv/presentation/ChannelCardPresenter.java

```diff
@@ -60,12 +42,6 @@ public void onBindViewHolder(Presenter.ViewHolder viewHolder, Object item) {
 
     @Override
     public void onUnbindViewHolder(Presenter.ViewHolder viewHolder) {
-        //Log.d(TAG, "onUnbindViewHolder");
+        // Unused
```

Comment:
Why keep this override if it is unused?

Reply:
It's abstract in the parent so it needs to be implemented